core/src/test/java/org/apache/jsieve/VacationTest.java

```diff
@@ -111,9 +111,10 @@ public void vacationShouldWork() throws Exception {
 
         JUnitUtils.interpret(sieveMailAdapter, script);
 
-        verify(sieveMailAdapter, times(2)).setContext(any(SieveContext.class));
+        verify(sieveMailAdapter).setContext(any(SieveContext.class));
+        verify(sieveMailAdapter).setContext(isNull(SieveContext.class));
         verify(sieveMailAdapter).addAction(ActionVacation.builder().reason("reason").build());
-        verify(sieveMailAdapter, times(2)).addAction(any(ActionKeep.class));
+        verify(sieveMailAdapter).addAction(any(ActionKeep.class));
```

Comment:
is there any reason for removing `times(2)` verification?

Reply:
Well honestly... I checked the code, and I didn't see how this was called twice, it seems to me it just gets called once... But I might be wrong. However with the mockito upgrade times(2) is wrong now. Also difference with `any()` in mockito is that before it was ok with null or empty params on v1, but not later versions. Not sure if it's because of that it was ok before but could be...

If someone wants to double check though, be my guest !